web-ui/src/main/resources/catalog/locales/en-core.json

```diff
@@ -475,7 +475,7 @@
     "metadataUnpublishedError": "Error occurred while un-publishing metadata.",
     "categoriesUpdated": "Categories updated.",
     "warnPublishDraft": "When publishing records with workflow enabled, the status will change to 'Approve'. Are you sure you want to continue?",
-    "canceWorkingCopy": "Cancel working copy",
+    "cancelWorkingCopy": "Cancel working copy",
```

Comment:
The "issue" is that when you change a key (or even a value), then all translations are reset on transifex side (and give some work on translators side). So not sure if we can leave with a typo on a key or not? :)

Reply:
It doesn't sound right to leave typos in the code to avoid translation. I'm not sure what cost is involved with the translation - maybe wait until other changes are required for these json files before approving the request so that transifex can get all the new changes in one batch?

Reply:
cost is pretty intense, i would wait for a major release...

Reply:
As a translator for french, it is quite painful to see translations reset by transifex on every key/value change in english even if it shows history to quickly set it back. If we reset now and make 3.8.2 in 10 days or so, then you have all translators to update that in this short time. This is more an issue on transifex and I don't see any option to avoid that. But I'm not a transifex expert...

The other option would be to have a transifex version for master and 3.8.x and then maintain the 2, but with the limited resources taking care of this so far, I would not go that way.

Reply:
Funny - transifex - web site claims fast - but don't mention that it is painful :)

I though that committing to the master would be for the next major release and committing to branch_3.8.x would be for the next 3.8 release - i.e. 3.8.2. (this pull request is for the master)

Either way - waiting for major release is ok with me. - just hope nobody forgets to approve the change before the next major release :)

Reply:
@pvgenuchten is it fine for you if we merge this PR (and Jo's one https://github.com/geonetwork/core-geonetwork/pull/4152, #4153, #4154, #4155) so translators will have one month to update the translation for the next coming release ?